client/lib/products-values/constants.js

```diff
@@ -45,6 +47,11 @@ export const isJetpackSearch = ( slug ) => JETPACK_SEARCH_PRODUCTS.includes( slu
 
 export const JETPACK_SCAN_PRODUCTS = [ PRODUCT_JETPACK_SCAN, PRODUCT_JETPACK_SCAN_MONTHLY ];
 
+export const JETPACK_ANTISPAM_PRODUCTS = [
+	PRODUCT_JETPACK_ANTISPAM,
+	PRODUCT_JETPACK_ANTISPAM_MONTHLY,
+];
```

Comment:
Same question here.

Reply:
+1

Reply:
Resolved in 9b90064b9c4ee3c3e26d380a29b4051ebc1328b2.